### DISCUSSION TITLE
What band released the album?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1005] [Galaxy Trivia] [answered]
"🤪? Yes! 🤤? No!"

**Comment by peterbaccaro** [comment]
The album "Crazy? Yes! Dumb? No!" was released by the New Zealand indie rock band **The Mint Chicks**.

**Comment by thuongtruong1009** [comment, marked as answer]
The band that released the album with the hint "from this icon 🤪? Yes! 🤤? No!" is Panic! at the Disco. The album referred to is "Pray for the Wicked," which was released by Panic! at the Disco in 2018.
